routes/celebrities.routes.js

```diff
@@ -0,0 +1,49 @@
+const CelebrityModel = require("../models/Celebrity.model");
+
+const router = require("express").Router();
+
+// all your routes here
+
+router.get("/celebrities", (req, res) => {
+    CelebrityModel.find()
+    .then((celebritiesDetails) => {
+        const data = {celebritiesDetails}
+        res.render('celebrities/celebrities',data)
+    })
+    .catch((error) => {
+        console.log("Error retrieving celebrities from the database. ", error);
+        next(error);
+    });
+})
+
+router.get("/celebrities/create", (req, res) => {
+    CelebrityModel.find()
+    .then((celebritiesDetails) => {
+        const data = {celebritiesDetails}
+        res.render('celebrities/new-celebrity',data)
+    })
+    .catch((error) => {
+        console.log("Error retrieving celebrities from DB, ", error)
+        next(error);
+    })
+
+});
```

Comment:
In the GET route where you have form to create some celebrity -> there is no need to access database! All you need to do is render the hbs with form ;)

```
router.get("/celebrities/create", (req, res) => {
     res.render('celebrities/new-celebrity')
});
```